### ISSUE TITLE
Bug report : Running the jar errors

### ISSUE THREAD

**Issue by DillonB07** [MommyHeather/AdvancedBackups#50]
**Describe the bug**

When I try running the mod, it errors and does nothing,

```sh
$ java -jar "AdvancedBackups-fabric-1.20-3.3.jar"
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/gson/JsonParseException
	at co.uk.mommyheather.advancedbackups.cli.AdvancedBackupsCLI.main(AdvancedBackupsCLI.java:46)
Caused by: java.lang.ClassNotFoundException: com.google.gson.JsonParseException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 1 more
```

**To Reproduce**
1. Make a server. Do stuff with incremental backups on.
2. Download the mod and the backup folder
3. Run the jar

**Expected behavior**
The menu for restoring a backup

**Screenshots**
<img width="655" alt="image" src="https://github.com/MommyHeather/AdvancedBackups/assets/83948303/07fef835-c6b7-4dad-9f86-118446d619a6">

**Versioning (please complete the following information):**
 - OS: [e.g. Debian]: Server: A form of Linux - I'm not sure what specifically. Local: macOS 12.7.1
 - Minecraft Version(s): [e.g 1.18, 1.12]: 1.20.1
 - Modloader(s): [e.g. Forge]: Fabric
 - Mod Version: [e.g 1.0]: 3.3

**Additional context**
Accidentally deleted something on my server so I downloaded the backups in hope of restoring it, however the backup tool refused to load.

I do not have ssh access to the server.

**Attempts to fix issue**
- Downloaded all server files except `world` and ran the mod from the mods folder
- Created a world folder and started the server to let it auto generate.

**Comment by MommyHeather** [comment]
This was fixed in version 3.4. You can get the relevant file on [curseforge](https://www.curseforge.com/minecraft/mc-mods/advanced-backups/files/4913299) or [modrinth](https://modrinth.com/mod/advanced-backups/version/9ts2YEHP).

Sticking this jar in place of the old one will let you run it directly. 

As a side note with the setup you have, it's worth noting that the tool expects a proper server-like or client-like setup - that is, it'll expect to be in a folder (usually named `mods`), and in the parent of that it'll check `config/AdvancedBackups.properties` for your backup path. You might want to make sure your local copy of the backups and mod is structured in the same way as the server itself.

Feel free to let me know if you have any further issues.

**Comment by DillonB07** [comment]
Oh, thanks! - I missed that update
